### PR TITLE
HARVESTER: Update Chinese translation

### PR DESCRIPTION
**Pull request by vickyhella**
HARVESTER: Update Chinese translation

### CODE REVIEW

**Comment by vickyhella** [comment]
@WuJun2016 @n313893254 Would you guys please take a look at the translation update? Thanks!

File: pkg/harvester/l10n/zh-hans.yaml [hunk 1]
```diff
@@ -286,8 +296,12 @@ harvester:
       run: 运行策略
       configure: 配置
       mode: 模式
-      modeLink: 模式 <a href="https://docs.harvesterhci.io/v1.0/host/host/#enable-ksmtuned" target="_blank"><i class="icon icon-info" /></a>
+      modeLink: 模式 <a href="https://docs.harvesterhci.io/next/host/#ksmtuned-mode" target="_blank"><i class="icon icon-info" /></a>
```
Comment (WuJun2016) [inline pkg/harvester/l10n/zh-hans.yaml:299]: use v1.1.0 release docs link

**Review by WuJun2016 — CHANGES_REQUESTED** [review]
other LGTM

**Comment by WuJun2016** [comment]
@n313893254   PTAL, thanks

File: pkg/harvester/l10n/zh-hans.yaml (hunk 1 above)
Comment (vickyhella) [reply pkg/harvester/l10n/zh-hans.yaml]: Already updated the link in both the English and Chinese files. The v1.1 doc links should be available once https://github.com/harvester/docs/pull/206 is merged.

**Review by n313893254 — APPROVED** [review]

**Pull request merged**